app/views/navbar.scala.html

```diff
@@ -179,69 +179,53 @@
         });
     }
 
-    // Triggered when 'Sign In' in navbar is clicked
-    // Logs "Click_module=SignIn_location=Navbar_route=</|/audit|/help|...>"
-    $("#sign-in-button").on('click', function(){
+    // The rest of this script logs whichever button is hit in the navbar.
+    // Example: Following logs "Click_module=PSLogo_location=Navbar_route=</|/audit|/help|...>".
+    $("#navbar-brand").on('click', function(){
```

Comment:
Can you add a space between the `)` and `{` on all of these, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide).